config/environments/development.rb

```diff
@@ -33,17 +44,23 @@
   config.eager_load = false
 
   require 'rack-webauth/test'
-  config.middleware.use(
-    Rack::Webauth::Test,
-    user: Settings.WEBAUTH.USER,
-    mail: Settings.WEBAUTH.MAIL,
-    ldapprivgroup: Settings.WEBAUTH.LDAPPRIVGROUP,
-    suaffiliation: Settings.WEBAUTH.SUAFFILIATION,
-    displayname: Settings.WEBAUTH.DISPLAYNAME,
-    ldapauthrule: Settings.WEBAUTH.LDAPAUTHRULE
-  )
+  if Settings.WEBAUTH
```

Comment:
Gotcha. Seems like we can nil out `Settings.WEBAUTH`

```ruby
Settings.WEBAUTH = nil
```

or we could just set this on the server?

```ruby
# in config/development.rb
WEBAUTH: nil
```

Thought both of these solutions seem strange to provide to essentially disable local development configurations. Happy to punt on this and come back to it.